src/server/myCoolService/myCoolService.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/bash
+
+# Yeah I did it.  I wrote my service in bash!
+
+
+
+echo "Your are awesome!"
```

Comment:
I would like to comment on this specific line ... Yes I know ... I am pretty 'awesome'